components/google-cloud/docs/source/index.rst

```diff
@@ -0,0 +1,20 @@
+.. google_cloud_pipeline_components documentation master file, created by
+   sphinx-quickstart on Fri May  7 04:23:22 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to google_cloud_pipeline_components's documentation!
+============================================================
```

Comment:
We may want to reference the python-aiplatform repo and docs here:

- https://github.com/googleapis/python-aiplatform
- https://googleapis.dev/python/aiplatform/latest/aiplatform.html#

Reply:
Also correcting in follow up PR.

Reply:
Addressed  in #5611